lib/msal-browser/src/event/EventHandler.ts

```diff
@@ -174,4 +181,26 @@ export class EventHandler {
             return;
         }
     }
+
+    private handleActiveAccountCacheChange(e: StorageEvent): void {
+        try {
+            const previousActiveAccountFilters = e.oldValue
+                ? JSON.parse(e.oldValue)
+                : null;
+            const newActiveAccountFilters = e.newValue
+                ? JSON.parse(e.newValue)
+                : null;
+
+            this.emitEvent(
+                EventType.ACTIVE_ACCOUNT_UPDATED,
+                undefined, // interactionType
```

Comment:
Do we care about the interactionType while emiting this?

Reply:
I don't believe so, `setActiveAccount` can be called from basically any scenario, I suspect most calls won't have an interaction type.

Reply:
+1 N/A